package.json

```diff
@@ -1,7 +1,7 @@
 {
-  "name": "fractalis",
+  "name": "@thehyve/fractalis",
   "description": "The front-end component of the Fractalis visual analytics project.",
-  "version": "1.4.0",
+  "version": "1.4.0-hyve-0.0.1",
```

Comment:
Is the suffix required if we have a different package name anyway?

Reply:
I thought it would be good to have a link between the original (base) fractalis version (1.4.0) and increment our version separately, but if this is not useful, I will remove the suffix.